scripts/jira-automation.py

```diff
@@ -333,6 +460,7 @@ def get_severity(self) -> str:
 
         return self.__severity
 
+
```

Comment:
shouldn't these getters be in the models.py?
Or maybe use OO techniques to model the objects?